src/vm/i386/excepx86.cpp

```diff
@@ -3769,10 +3783,27 @@ AdjustContextForVirtualStub(
     return TRUE;
 }
 
+#ifndef DACCESS_COMPILE
+#ifndef WIN64EXCEPTIONS
 #ifdef FEATURE_PAL
+EXTERN_C
+_Unwind_Reason_Code
+UnhandledExceptionHandlerUnix(
```

Comment:
I am sorry for not being clear. That is exactly what I meant - once we enable WIN64EXCEPTIONS for x86 Linux, this stub will be eliminated. So that's why I was wondering what is the reason for moving it from one file to another when it will be removed soon anyways.